doc/readme.html

```diff
@@ -545,6 +545,8 @@ <h4>Optional interfaces to other software packages</h4>
                 <p>
                     <a href="https://github.com/kokkos/kokkos" target="_top">Kokkos</a> implements a programming model in C++ for writing performance portable applications targeting all major HPC platforms.
                     To use a self compiled version, pass <code>-DKOKKOS_DIR=/path/to/kokkos</code> to the deal.II CMake call.
+                    The compiler must be able to compile code for all enabled backends. In case Cuda is enabled in Kokkos and nvcc should be used as device compiler,
+                    it is recommended to use the nvcc_wrapper script that comes with Kokkos as C++ compiler. clang++ or nvc++ (for Kokkos >= 4.0) could be used directly as C++ compiler.
```

Comment:
Should we add a comment that if people use CUDA, then need to compile Kokkos with `Kokkos_ENABLE_CUDA_LAMBDA=ON`? I don't want to write functors everywhere.

Reply:
Yes, and we should then check that this is indeed set, right?

Reply:
See https://github.com/dealii/dealii/blob/9cf00ca5f0aa42bc54540c61c60af0222804c15e/cmake/modules/FindKOKKOS.cmake#L55-L59 for an external `Kokkos` and https://github.com/dealii/dealii/pull/14435 for `Kokkos` bundled with `Trilinos`.

Reply:
I added a comment in [`4fbe1bb` (#14365)](https://github.com/dealii/dealii/pull/14365/commits/4fbe1bbead29043d75a908881d9496bf7f85fe7d).